pokemon-app/src/styles/CreditScreen.css

```diff
@@ -87,7 +88,7 @@
     height: 23px;
     margin-top: -2px;
 }
-.whole-noct{
+.wholeNoct{
```

Comment:
Why did you move away from the hyphen case?

Reply:
I noticed I had named it without a hyphen in my earlier code haha

Reply:
I think the hyphen case is better, or at least for consistency's sake.  At BWTC, we only use the hyphen case in css.

Reply:
Ok, thanks Kendrick! I'll make sure to stick to hyphens from now on.